tchannel/testing/vcr/patch.py

```diff
@@ -110,12 +118,13 @@ def send(self, arg1, arg2, arg3,
                     e.message
                 )
             )
+
         response = Response(
-            code=vcr_response.code,
+            code=vcr_response.status,
```

Comment:
(the code we want here is the recorded status code, not the status code of making this call to the VCR service)